nodejs/week3/src/warmUp3/index3.js

```diff
@@ -0,0 +1,41 @@
+const express = require("express");
+const app = express();
+const PORT = 3000;
+
+app.use(express.json());
+app.use(
+  express.urlencoded({
+    extended: true,
+  })
+);
+
+// Calculator using req.body
+
+app.get("/calculator", (req, res) => {
+  const method = req.body.method;
+  const arrayOfNumbers = Object.values(req.query);
```

Comment:
This is smart! Yey! :)

The specification in the exercise says that only two params are allowed `firstParam` and `secondParam`. This would work even with any arbitrary param `cucumber`, `loafOfBread` as well.

In this example I'm going to call that a feature instead of a bug :)